backend/app/services/adapters/executor_kinds.py

```
@@ -1955,6 +1963,26 @@ def _create_next_pipeline_stage_subtask(
+        try:
+            from app.services.task_dispatcher import task_dispatcher
+
+            if task_dispatcher.enabled:
+                # Dispatch the pending subtask for this specific task
+                import asyncio
+
+                asyncio.create_task(
+                    task_dispatcher.dispatch_pending_tasks(
+                        db, task_ids=[task.id], limit=1
+                    )
+                )
```

Ruff (0.14.13)

Store a reference to the return value of `asyncio.create_task`

(RUF006)